### PR TITLE
 Move Update Modules section to Board Integration.

### PR DESCRIPTION
**Pull request by kacf**

### CODE REVIEW

**Comment by kacf** [comment]
@estenberg: There are two references to the existing doc location in our blog. Perhaps you want to update those and merge this at the same time?

File: 03.Devices/10.Update-Modules/docs.md [hunk 1]
```diff
@@ -6,9 +6,11 @@ taxonomy:
 
 ## Introduction
 
-An _Update Module_ is an executable that extends the Mender client to support new types of software updates, such as package managers, containers, bootloader updates and even nearby microcontrollers. An Update Module can be tailored to a specific device or environment (e.g. update a proprietary bootloader), or be more general-purpose (e.g. install a set of `.deb` packages.).
+The built-in update mechanism is Mender is [dual rootfs updates](../../architecture/overview#robust-updates). To do other types of updates, you will need an _Update module_ for your device.
```
Comment (estenberg) [inline 03.Devices/10.Update-Modules/docs.md]: ...mechanism *in* Mender..

**Comment by estenberg** [comment]
Yes, will do. Just a minor comment.

**Comment by kacf** [comment]
@estenberg: Thanks! Merge whenever you want!

**Comment by kacf** [comment]
And this one too: #665.

**Comment by estenberg** [comment]
Great, thank you!
Blog updated.

**Pull request merged**